### ISSUE TITLE
required propTypes

### ISSUE THREAD

**Issue by flynfish** [rkit/react-select2-wrapper#12]
Right now the `defaultValue` is required to be number or array. A string is perfectly valid when the id is not a number. Can you add the string as a valid value?

```javascript
defaultValue: PropTypes.oneOfType([
      PropTypes.number,
      PropTypes.array,
      PropTypes.string,
    ])
```

**Comment by rkit** [comment]
Ok, Today will be release.

**Comment by rkit** [comment]
I released a new version (0.3.0)

**Comment by flynfish** [comment]
thanks!
